demo/site/src/common/blocks/AccordionItemBlock.tsx

```diff
@@ -98,6 +98,15 @@ const ContentWrapper = styled.div<{ $isExpanded: boolean }>`
         `}
 `;
 
-const ContentWrapperInner = styled.div`
+const ContentWrapperInner = styled.div<{ $isExpanded: boolean }>`
     overflow: hidden;
+    visibility: hidden;
+    transition: visibility 350ms;
+    transition-delay: ${({ $isExpanded }) => ($isExpanded ? "0ms" : "350ms")};
```

Comment:
Do we need this additional animation? We already "animate" grid-template-rows in ContentWrapper.

Reply:
I removed the animation time, but the transition itself is needed to delay the visibility. Otherwise the content is immediately hidden and the content is empty while collapsing.